recipes/geomet/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "geomet" %}
+{% set version = "0.2.1.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 91d754f7c298cbfcabd3befdb69c641c27fe75e808b27aa55028605761d17e95
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - geomet = geomet.tool:cli
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six
+    - click
+
+test:
+  imports:
+    - geomet
```

Comment:
If possible please test the cli:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
@synapticarbors  yes, done.
Thanks for the hint